src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,32 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { merge } from '@utilities/merge';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { ReactElement } from 'react';
+import { ToastProps, toastStylesBackgroundColorsMap } from './types';
+
+export const Toast = ({ isOpen, style, icon, children }: ToastProps): ReactElement => (
+    <AnimatePresence>
+        {isOpen && (
+            <motion.div
+                className="tw-min-h-[4.25rem] tw-rounded-lg tw-border tw-shadow-lg tw-bg-white tw-border-black-10"
```

Comment:
As all our overlays, you can use the `CONTAINER_CLASSES` like in the `Menu` Component.